src/mono/netcore/sample/iOS/HelloiOS/runtime.m

```diff
@@ -0,0 +1,377 @@
+#import <Foundation/Foundation.h>
+#include <mono/utils/mono-publib.h>
+#include <mono/utils/mono-logger.h>
+#include <mono/metadata/assembly.h>
+#include <mono/metadata/mono-debug.h>
+#include <mono/metadata/exception.h>
+#include <mono/jit/jit.h>
+
+#import <os/log.h>
+#include <sys/stat.h>
+#include <sys/mman.h>
+
+static os_log_t stdout_log;
+
+/* These are not in public headers */
+typedef unsigned char* (*MonoLoadAotDataFunc) (MonoAssembly *assembly, int size, void *user_data, void **out_handle);
```

Comment:
@lambdageek could this be fixed?

Reply:
Yes let's file an issue about this, we shouldn't need to rely on private functions.